lib/opal/sprockets/server.rb

```diff
@@ -4,7 +4,14 @@
 require 'rack/builder'
 require 'rack/deflater'
 require 'rack/directory'
-require 'rack/showexceptions'
+
+# rack changed some paths in 2.0.0alpha
+begin
+  require 'rack/showexceptions'
+rescue LoadError
+  require 'rack/show_exceptions'
+end
+
```

Comment:
I see now. I think it's fine to `require 'rack'` since it uses autoload and doesn't require anything else… 

thanks to this I just discovered that since it uses autoload the requires can be skipped altogether and the `ShowExceptions` and `ConditionalGet` by virtue of autoload :D